templates/teedy-docs.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Teedy-docs</Name>
+  <Repository>sismics/docs:latest</Repository>
+  <Registry>https://hub.docker.com/r/sismics/docs/</Registry>
+  <Branch>
+  <Tag>latest</Tag>
+  <TagDescription>Master branch, can be unstable. Not recommended for production use</TagDescription>
+  </Branch>
+  <Branch>
+  <Tag>v1.8</Tag>
+  <TagDescription>Latest stabel version</TagDescription>
+  </Branch>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://hub.docker.com/r/sismics/docs/</Support>
+  <Project>https://github.com/sismics/docs</Project>
+  <Overview>Teedy is an open-source, lightweight document management system (DMS) for individuals and businesses.&#xD;
+&#xD;
+The default admin password is "admin". Don't forget to change it before going to production.</Overview>
+  <Category>Cloud: Productivity:</Category>
+  <WebUI>http://[IP]:[PORT:8080]</WebUI>
+  <Icon>https://github.com/sismics/docs/blob/master/docs-web/src/main/resources/image/logo.png?raw=true</Icon>
+  <Description>Teedy is an open-source, lightweight document management system (DMS) for individuals and businesses.&#xD;
+&#xD;
+The default admin password is "admin". Don't forget to change it before going to production.</Description>
+  <Config Name="WebUI Port" Target="8080" Default="" Mode="tcp" Description="Container Port: 8080" Type="Port" Display="always" Required="true" Mask="false">8356</Config>
+  <Config Name="Data Storage" Target="/data" Default="/mnt/user/appdata/Teedy" Mode="rw" Description="Container Path: /data" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/Teedy</Config>
```

Comment:
You can do more shaving here, such as removing the actual data, and keep the tags.

Reply:
you mean like this?
```
<Config Name="WebUI Port" Target="8080" Default="" Mode="tcp" Description="Container Port: 8080" Type="Port" Display="always" Required="true" Mask="false"/>
<Config Name="Data Storage" Target="/data" Default="/mnt/user/appdata/Teedy" Mode="rw" Description="Container Path: /data" Type="Path" Display="always" Required="true" Mask="false"/>
```